tests/main/classic-prepare-image/task.yaml

```diff
@@ -52,7 +52,7 @@ restore: |
     systemctl stop snapd.service snapd.socket fakedevicesvc
 
     rm -rf "$SEED_DIR"
-    systemctl start snapd.socket snapd.service
+    systemctl start snapd.socket
```

Comment:
I think I'd move this after the fake store is torn down, and "$ROOT" is removed. Also, maybe worth checking if we really need to start snapd in restore. The reset code will stop it anyway.

Reply:
Plus we need to apply this to both task.yaml files, right? It seems the other has very similar code and there is no change.